src/hooks.ts

```diff
@@ -162,3 +170,48 @@ export function usePointPosition(config: UsePositionConfig[]) {
     )
     .join(' ');
 }
+interface UseShiftOptions extends DualAxisOptions {
+  xShift?: number | string;
+  yShift?: number | string;
+}
+export function useShift(options: UseShiftOptions) {
+  const { axisContext, plotWidth, plotHeight } = usePlotContext();
+  const {
+    horizontalAxisId = 'x',
+    verticalAxisId = 'y',
+    xShift = '0',
+    yShift = '0',
+  } = options;
+  const [xScale, yScale] = validateAxis(
+    axisContext,
+    horizontalAxisId,
+    verticalAxisId,
+  );
+  return {
+    xShift: convertToPx(xShift, plotWidth, xScale),
+    yShift: convertToPx(yShift, plotHeight, yScale),
+  };
+}
+
+interface UseInvertOptions extends DualAxisOptions {
+  xShift?: number;
+  yShift?: number;
+}
+export function useInvertShift(options: UseInvertOptions) {
+  const { axisContext } = usePlotContext();
+  const {
+    horizontalAxisId = 'x',
+    verticalAxisId = 'y',
+    xShift = 0,
+    yShift = 0,
+  } = options;
+  const [xScale, yScale] = validateAxis(
+    axisContext,
+    horizontalAxisId,
+    verticalAxisId,
+  );
+  return {
+    xShift: toNumber(xScale?.invert(xShift)) - toNumber(xScale?.invert(0)),
+    yShift: toNumber(yScale?.invert(0)) - toNumber(yScale?.invert(yShift)),
```

Comment:
If you are inverting the sign for `y` in places which use this hook, wouldn't it make more sense to just use the same calculation here as for `xShift` which will also invert the sign?

Reply:
problem fixed thank you